### DISCUSSION TITLE
isSubmitSuccessful remains false after submitting form

### DISCUSSION THREAD

**Discussion by ChronoLoop** [react-hook-form/react-hook-form#7459] [Q&A] [answered]
https://codesandbox.io/s/react-hook-form-handlesubmit-with-reset-forked-z1dxo?file=/src/index.js:765-783

If I comment out `reset(submittedData); `, `isSubmitSuccessful` turns `true`. I am guessing it is because it is not the same reference. Is there an option to keep `isSubmitSuccessful` when calling `reset`?

**Comment by bluebill1049** [comment, marked as answer]
not at the moment, you can submit feature requests and see if it's a common demand in the community.

**Reply by ChronoLoop** [reply]
Thanks for the response
https://github.com/react-hook-form/react-hook-form/issues/7464
